packages/components/src/LightBox/LightBox.mdx

```diff
@@ -0,0 +1,74 @@
+---
+name: LightBox
+menu: Components
+route: /components/light-box
+---
+
+import { useState } from "react";
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Button } from "../Button";
+import { LightBox } from ".";
+
+# Light Box
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+LightBox is a component designed to display an array of images to the user. A
+user clicks on a control and is shown a carousel style presentation of the
+defined images. This component is a wrapper of the frontend-collective's
+react-image-lightbox third party library.
+
+```ts
+import { LightBox } from "@jobber/components/LightBox";
+```
+
+<Playground>
+  {() => {
+    const [open, isOpen] = useState(false);
+    return (
+      <>
+        <Button label="😈 Click me!" onClick={() => isOpen(true)} />
+        <LightBox
+          open={open}
+          images={[
+            {
+              title: "😃",
+              url: "https://i.imgur.com/6Jcfgnp.jpg",
+              caption: "Happy face"
+            },
+            {
+              title: "😮",
+              url: "https://i.imgur.com/OsLxYkT.jpg",
+              caption: "Suprised face"
+            },
+            {
+              url: "https://i.imgur.com/uigfzs5.jpg"
+            }
+          ]}
+          onRequestClose={() => isOpen(false)}
+        />
+      </>
+    );
+  }}
+</Playground>
+
+## Design & Usage Guidelines
+
+When contributing to, or consuming the LightBox component, consider the
```

Comment:
This section is meant to be more usage oriented a good example of such would be the Best Practices / Content Guidelines here:

https://polaris.shopify.com/components/overlays/popover#navigation